src/Data/Items.cs

```diff
@@ -285,19 +286,44 @@ public class ItemLookup {
             { "Ladders in Lower Quarry", new ItemData("Ladders in Lower Quarry", "progression", "Ladders in Lower Quarry", ItemTypes.LADDER, 1) }, // DONE
             { "Ladders in Library", new ItemData("Ladders in Library", "progression", "Ladders in Library", ItemTypes.LADDER, 1) }, // DONE
 
+            // Fuses
+            { "Swamp Fuse 1", new ItemData("Swamp Fuse 1", "progression", "Swamp Fuse 1", ItemTypes.FUSE, 1) },
+            { "Swamp Fuse 2", new ItemData("Swamp Fuse 2", "progression", "Swamp Fuse 2", ItemTypes.FUSE, 1) },
+            { "Swamp Fuse 3", new ItemData("Swamp Fuse 3", "progression", "Swamp Fuse 3", ItemTypes.FUSE, 1) },
+            { "Cathedral Elevator Fuse", new ItemData("Cathedral Elevator Fuse", "progression", "Cathedral Elevator Fuse", ItemTypes.FUSE, 1) },
+            { "Quarry Fuse 1", new ItemData("Quarry Fuse 1", "progression", "Quarry Fuse 1", ItemTypes.FUSE, 1) },
+            { "Quarry Fuse 2", new ItemData("Quarry Fuse 2", "progression", "Quarry Fuse 2", ItemTypes.FUSE, 1) },
+            { "Ziggurat Miniboss Fuse", new ItemData("Ziggurat Miniboss Fuse", "progression", "Ziggurat Miniboss Fuse", ItemTypes.FUSE, 1) },
+            { "Ziggurat Teleporter Fuse", new ItemData("Ziggurat Teleporter Fuse", "progression", "Ziggurat Teleporter Fuse", ItemTypes.FUSE, 1) },
+            { "Fortress Exterior Fuse 1", new ItemData("Fortress Exterior Fuse 1", "progression", "Fortress Exterior Fuse 1", ItemTypes.FUSE, 1) },
+            { "Fortress Exterior Fuse 2", new ItemData("Fortress Exterior Fuse 2", "progression", "Fortress Exterior Fuse 2", ItemTypes.FUSE, 1) },
+            { "Fortress Courtyard Upper Fuse", new ItemData("Fortress Courtyard Upper Fuse", "progression", "Fortress Courtyard Upper Fuse", ItemTypes.FUSE, 1) },
+            { "Fortress Courtyard Fuse", new ItemData("Fortress Courtyard Fuse", "progression", "Fortress Courtyard Fuse", ItemTypes.FUSE, 1) },
+            { "Beneath the Vault Fuse", new ItemData("Beneath the Vault Fuse", "progression", "Beneath the Vault Fuse", ItemTypes.FUSE, 1) },
+            { "Fortress Candles Fuse", new ItemData("Fortress Candles Fuse", "progression", "Fortress Candles Fuse", ItemTypes.FUSE, 1) },
+            { "Fortress Door Left Fuse", new ItemData("Fortress Door Left Fuse", "progression", "Fortress Door Left Fuse", ItemTypes.FUSE, 1) },
+            { "Fortress Door Right Fuse", new ItemData("Fortress Door Right Fuse", "progression", "Fortress Door Right Fuse", ItemTypes.FUSE, 1) },
+            { "Furnace to Well Fuse", new ItemData("Furnace to Well Fuse", "progression", "Furnace to Well Fuse", ItemTypes.FUSE, 1) },
```

Comment:
Maybe we should just call this one "Furnace Fuse"?

Reply:
We already have a region called that in the logic so I wanted to avoid the duplicate or having to rename it

Reply:
I think I'd prefer renaming the region in that case o: "Furnace Fuse" is just a better item name, and that actually gets displayed to players unlike region names (usually)

Reply:
How about "West Furnace Fuse"

Reply:
That works o: